dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/preheat/TrackerPreheat.java

```diff
@@ -631,18 +632,17 @@ public Optional<TrackedEntityComment> getNote( String uid )
         return Optional.ofNullable( notes.get( uid ) );
     }
 
-    public Relationship getRelationship( org.hisp.dhis.tracker.domain.Relationship relationship )
+    public Relationship getRelationship( String relationshipUid )
```

Comment:
did you have to make this param change?

if so, I would still leave a `public Relationship getRelationship( org.hisp.dhis.tracker.domain.Relationship relationship )` in there that then calls `getRelationship( rel.getRelationship() )`

this would client code nicer to read

Reply:
I added the method